setup/inject_SELinux_AVC_check/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+BEAKERLIB_SCRIPT=/usr/share/beakerlib/logging.sh
+
+rlJournalStart
+
+    rlPhaseStartTest
+        rlLogInfo "Injecting the test code into rlPhaseStartTest"
+        rlRun "cp $BEAKERLIB_SCRIPT $BEAKERLIB_SCRIPT.pre_injection_backup" 0 "Making backup of $BEAKERLIB_SCRIPT"
```

Comment:
use `$BEAKERLIB_SCRIPT.pre_injection_backup.$$` to make the name a bit more unique..just for sure.